helper/validation/strings.go

```diff
@@ -174,41 +176,56 @@ func StringIsBase64(i interface{}, k string) (warnings []string, errors []error)
 
 	// NoEmptyStrings checks it is a string
 	v, _ := i.(string)
+
 	if _, err := base64.StdEncoding.DecodeString(v); err != nil {
 		errors = append(errors, fmt.Errorf("expected %q to be a base64 string, got %v", k, v))
 	}
 
-	return
-}
-
-// ValidateListUniqueStrings is a ValidateFunc that ensures a list has no
-// duplicate items in it. It's useful for when a list is needed over a set
-// because order matters, yet the items still need to be unique.
-func ValidateListUniqueStrings(v interface{}, k string) (ws []string, errors []error) {
-	for n1, v1 := range v.([]interface{}) {
-		for n2, v2 := range v.([]interface{}) {
-			if v1.(string) == v2.(string) && n1 != n2 {
-				errors = append(errors, fmt.Errorf("%q: duplicate entry - %s", k, v1.(string)))
-			}
-		}
-	}
-	return
+	return warnings, errors
 }
 
 // ValidateJsonString is a SchemaValidateFunc which tests to make sure the
 // supplied string is valid JSON.
-func ValidateJsonString(v interface{}, k string) (ws []string, errors []error) {
+//
+// Deprecated: use StringIsJSON instead
+func ValidateJsonString(i interface{}, k string) (warnings []string, errors []error) {
+	return StringIsJSON(i, k)
+}
+
+// ValidateJsonString is a SchemaValidateFunc which tests to make sure the supplied string is valid JSON.
+func StringIsJSON(i interface{}, k string) (warnings []string, errors []error) {
```

Comment:
😨 This is late in the process apologies. Do we want to call this `IsJSONString`? To stay aligned with functions starting with `Is`?